blocks/api/test/serializer.js

```diff
@@ -195,13 +199,14 @@ describe( 'block serializer', () => {
 				{
 					name: 'core/test-block',
 					attributes: {
+						foo: false,
```

Comment:
One of two things should be done here:

- We should set a `bar: false` value in the attributes here. As is we're not really testing that the default value is omitted, because the attribute isn't set anyways.
- We should use `createBlock` to construct the block object and rely on its behavior to set the defaults into attributes.

My preference is toward the latter.